src/attributeFields.js

```diff
@@ -24,7 +25,18 @@ module.exports = function (Model, options = {}) {
     };
 
     if (memo[key].type instanceof GraphQLEnumType ) {
-      memo[key].type.name = `${Model.name}${key}EnumType`;
+      var typeName = `${Model.name}${key}EnumType`;
+      // Check cache for existing type
+      if (cache[typeName]) {
+        // Use cached type
+        // Prevent GraphQL error message
+        memo[key].type = cache[typeName];
+      } else {
+        // Not found in cache
```

Comment:
You can remove the multiple comments, and just do one comment ala 'Cache enum types to prevent duplicate type name error when calling attributeFields multiple times on the same model'